examples/profile.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE>
+<html ng-app='app'>
+  <head>
+    <title>Interaction button group example</title>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="initial-scale=1.0, user-scalable=no, width=device-width">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel="stylesheet" href="../node_modules/openlayers/css/ol.css" type="text/css">
+    <link rel="stylesheet" href="http://getbootstrap.com/dist/css/bootstrap.min.css" type="text/css">
+    <style>
+
+        .profile {
+          width: 450px;
+          height: 250px;
+        }
+
+    </style>
+  </head>
+  <body ng-controller="MainController as ctrl">
+    <div class="profile" ngeo-profile="ctrl.profileData"
+         ngeo-profile-on-hover="ctrl.onProfileHover(point)"
+         ngeo-profile-on-out="ctrl.onProfileOut()"></div>
```

Comment:
I'd use `onhover` and `onout` (no space) as we do anywhere else. In this way we always have one dash (two words) for the directive itself and two dashes (three words) for directive attributes.